posthog/models/organization.py

```diff
@@ -121,7 +121,7 @@ def _billing_plan_details(self) -> Tuple[Optional[str], Optional[str]]:
     def billing_plan(self) -> Optional[str]:
         return self._billing_plan_details[0]
 
-    def update_available_features(self) -> List[AvailableFeature]:
+    def update_available_features(self) -> List[Union[AvailableFeature, str]]:
```

Comment:
How come this accepts `str`? Wouldn't we just want to support valid enum keys?

Reply:
We _would_, but then in tests we try things like `"whatever"` out, which I didn't care enough about to change now.